lib/spack/spack/compiler.py

```diff
@@ -152,6 +251,53 @@ def check(exe):
     def version(self):
         return self.spec.version
 
+    @classmethod
+    def verbose_flag(cls):
+        """
+        This property should be overridden in the compiler subclass if a
+        verbose flag is available.
+
+        If it is not overridden, it is assumed to not be supported.
+        """
+        pass
+
+    @classmethod
+    def parse_implicit_rpaths(cls, string):
```

Comment:
Just remove this classmethod and make `_parse_implicit_rpaths` non-private.

Reply:
This is so that someone has a hook to add a compiler that we don't know about if its format isn't properly parsed by the overall method. Are you sure we want to get rid of that?

Reply:
good point.  No -- let's leave it.